src/Markdig/Extensions/Alerts/AlertBlockRenderer.cs

```diff
@@ -61,7 +61,9 @@ protected override void Write(HtmlRenderer renderer, AlertBlock obj)
     /// <param name="kind">The kind of the alert to render</param>
     public static void DefaultRenderKind(HtmlRenderer renderer, StringSlice kind)
     {
-        string? html = kind.AsSpan() switch
+        Span<char> upperKind = stackalloc char[kind.Length];
```

Comment:
This method might be better protected by checking e.g `kind.Length >= 16` and exit before trying to stackalloc.